src/os/bluestore/BlueStore.cc

```diff
@@ -5380,7 +5406,12 @@ int BlueStore::_open_db(bool create, bool to_repair_db, bool read_only)
                                  " \t");
     for (auto& i : cf_map) {
       dout(10) << "column family " << i.first << ": " << i.second << dendl;
-      cfs.push_back(KeyValueDB::ColumnFamily(i.first, i.second));
+      bool share_cache = true;
```

Comment:
does this have any implications for reading DBs with the old set of column families?